### PR TITLE
Avoid broken beaker versions 4.22.0 and 4.23.0

### PR DESCRIPTION
**Pull request by dhoppe**
* https://travis-ci.org/github/voxpupuli/puppet-kafka/jobs/689172715 # failure (beaker 4.23.0)
* https://travis-ci.org/github/voxpupuli/puppet-kafka/jobs/689178429 # success (beaker 4.22.1)

### CODE REVIEW

**Comment by ekohl** [comment]
https://github.com/puppetlabs/beaker/pull/1644#issuecomment-631394461

Let's hold off a bit and hope for a fixed 4.23.1.

File: voxpupuli-acceptance.gemspec [hunk 1]
```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |s|
 
   # Testing
   s.add_runtime_dependency 'bcrypt_pbkdf'
-  s.add_runtime_dependency 'beaker', '>= 4.2.0'
+  s.add_runtime_dependency 'beaker', '4.22.1'
```
Comment (ekohl) [inline voxpupuli-acceptance.gemspec:17]: I think you can also use a more free syntax, hoping 4.23.1 fixes it.
```suggestion
  s.add_runtime_dependency 'beaker', '>= 4.2.0', '!= 4.22.0', '!= 4.23.0'
```

See https://guides.rubygems.org/patterns/#declaring-dependencies

**Comment by dhoppe** [comment]
> I'd prefer to squash these commits

I know and will take care of it, as soon as Travis CI has done his job.

**Comment by ekohl** [comment]
Thanks!

**Pull request merged**